### PR TITLE
Portfolio: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by emirsagit**
### Portfolio: setup and mobile version skeleton

- Added readme and linter
- Created mobile version of header part of 'portfolio template1'
- Styled by using flexbox

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        <nav>
+            <div class="nav-flex">
+                <a href="#" class="nav-text">My Logo</a>
+                <button class="nav-button" type="button">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" stroke="#6070FF" viewBox="0 0 24 24">
+                        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+                    </svg>
+                </button>
+            </div>
+        </nav>
```
Comment (mricanho) [inline index.html:21]: - [x] The header tag must wrap this section and no more

File: assets/css/app.css [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #172b4d;
+  margin-bottom: 0;
+}
+
+.bg-header {
+  background-image: url('../img/bg-header.png');
```
Comment (mricanho) [inline assets/css/app.css:105]: - [x] Your background looks different than the original design, please fix it

File: index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        <div class="header bg-header">
+            <h1 class="header-title">
+                I’m Mariane
```
Comment (mricanho) [inline index.html]: - [x] [OPTIONAL] Put your own name, it's your personal portfolio

File: index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                <p class="social-title">LET’S CONNECT</p>
+                <div class="social-icons">
+                    <img src="./assets/img/icons/vector1.svg" alt="twitter" class="social-icon-item">
+                    <img src="./assets/img/icons/vector2.svg" alt="linkedin" class="social-icon-item">
+                    <img src="./assets/img/icons/vector3.svg" alt="Microverse" class="social-icon-item">
+                    <img src="./assets/img/icons/vector4.svg" alt="github" class="social-icon-item">
+                    <img src="./assets/img/icons/vector5.svg" alt="git" class="social-icon-item">
+                </div>
+            </div>
+        </div>
```
Comment (mricanho) [inline index.html]: - [x] Instead of a "div" tag, this should be wrapped by a "section" tag

File: index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+            </p>
+            <div class="social">
+                <p class="social-title">LET’S CONNECT</p>
+                <div class="social-icons">
+                    <img src="./assets/img/icons/vector1.svg" alt="twitter" class="social-icon-item">
+                    <img src="./assets/img/icons/vector2.svg" alt="linkedin" class="social-icon-item">
+                    <img src="./assets/img/icons/vector3.svg" alt="Microverse" class="social-icon-item">
+                    <img src="./assets/img/icons/vector4.svg" alt="github" class="social-icon-item">
+                    <img src="./assets/img/icons/vector5.svg" alt="git" class="social-icon-item">
+                </div>
```
Comment (mricanho) [inline index.html]: - [x] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag), please fix it

File: index.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+        <div class="header bg-header">
+            <h1 class="header-title">
+                I’m Mariane
+                Glad to see you!
+            </h1>
+            <p class="header-text">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <div class="social">
+                <p class="social-title">LET’S CONNECT</p>
```
Comment (mricanho) [inline index.html]: - [x] The button is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design), please fix it

File: README.md [hunk 7]
```diff
@@ -1,45 +1,22 @@
...
-
-### Usage
-
-### Run tests
-
-### Deployment
-
+``` git clone https://github.com/emirsagit/Review-Past-Project.git ```
 
+``` cd Review-Past-Project ```
```
Comment (mricanho) [inline README.md]: - [x] [OPTIONAL] This is from your past project, please fix it

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @emirsagit, 

Good job so far! 🔥🔥 You work hard and it pays. Your README file looks professional, and you use CSS like a pro, correct use of gitflow.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] From now on, you need to commit more often, two or three commits are not a good practice. And remember that you need to write them in past-tense and imperative style, check [this link](https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages).

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

yes you got this

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (emirsagit) [reply index.html]: thanks

File: index.html (hunk 1 above)
Comment (DeliceLydia) [reply index.html:21]: Well addressed the issue

File: assets/css/app.css (hunk 2 above)
Comment (DeliceLydia) [reply assets/css/app.css:105]: Your background now looks similar to the design well done!

File: index.html (hunk 3 above)
Comment (DeliceLydia) [reply index.html]: Well  done!

File: index.html (hunk 4 above)
Comment (DeliceLydia) [reply index.html]: Well addressed the issue!

File: index.html (hunk 5 above)
Comment (DeliceLydia) [reply index.html]: Solved the issue!

File: index.html (hunk 6 above)
Comment (DeliceLydia) [reply index.html]: Well done!

File: README.md (hunk 7 above)
Comment (DeliceLydia) [reply README.md]: Good job on fixing this as well

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved 🎉🎉

Hi @emirsagit  

Your project is complete!  There is nothing else to say other than... it's time to merge it 
Congratulations! 🎉 


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**